configure-store-limit.md

```diff
@@ -72,4 +72,6 @@ store limit 1 5 remove-peer         // 设置 store 1 删除 peer 的速度上
 
 当 [`store-limit-version`](/pd-configuration-file.md#store-limit-version-从-v710-版本开始引入) 设置为 `v2` 时，Store Limit v2 生效。在此模式下，Operator 调度限制将根据 TiKV Snapshot 执行情况进行动态调整。当 TiKV 积压的任务较少时，PD 会增加其调度任务。相反，PD 会减少对该节点的调度任务。此时，你无需关注如何设置 `store limit` 以加快调度进度。
 
-在该模式下，TiKV 执行速度成为迁移进度的主要瓶颈。你可以通过 **TiKV Details** > **Snapshot** > **Snapshot Speed** 面板判断当前调度速度是否达到 TiKV 限流设置。通过调整 TiKV Snapshot Limit ([`snap-io-max-bytes-per-sec`](/tikv-configuration-file.md#snap-io-max-bytes-per-sec)) 来增加或减少该节点的调度速度。
\ No newline at end of file
+在该模式下，TiKV 执行速度成为迁移进度的主要瓶颈。你可以通过 **TiKV Details** > **Snapshot** > **Snapshot Speed** 面板判断当前调度速度是否达到 TiKV 限流设置。通过调整 TiKV Snapshot Limit ([`snap-io-max-bytes-per-sec`](/tikv-configuration-file.md#snap-io-max-bytes-per-sec)) 来增加或减少该节点的调度速度。
+
+在当前版本中，将该配置项设置为 `"v2"` 为实验特性，不建议在生产环境中使用。
```

Comment:
from https://github.com/pingcap/docs-cn/blob/148b399f97d5a9abec18733b0303b510297553c2/pd-configuration-file.md?plain=1#L430

Reply:
> This feature is experimental between 7.1 and 8.5, and will GA in 8.5

@lhy1024 I recommend:

1. Changing the base of this PR to `release-8.4`. I will cherry-pick this PR to v7.1 and later versions.
2. Creating a new PR to upadte the `master` branch for v8.5 changes: removing the experimental note in https://github.com/pingcap/docs-cn/blob/3c0975ec627e0328664da94daf5efda37d6fb562/pd-control.md?plain=1#L457